src/test/java/org/openrewrite/kotlin/KotlinTypeMappingTest.java

```diff
@@ -62,6 +77,17 @@ void extendsKotlinAny() {
         assertThat(goatType.getSupertype().getFullyQualifiedName()).isEqualTo("kotlin.Any");
     }
 
+    @Test
+    void fieldType() {
+        J.VariableDeclarations.NamedVariable variable = getField("field").getVariables().get(0);
+        J.Identifier id = variable.getName();
+        assertThat(variable.getType()).isEqualTo(id.getType());
```

Comment:
Info on test update:
The variable and name should have the same type
The fieldType should be a JavaType.Variable.
Updated to check the field type and type signatures.